x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/SpecBaseIntegrationTestCase.java

```diff
@@ -122,9 +122,15 @@ protected boolean logEsResultSet() {
         return false;
     }
 
-    protected void assertResults(ResultSet expected, ResultSet elastic) throws SQLException {
+    /*
+     * Asserts the numeric results for floating point numbers in a leninent way, if chosen to. Usually,
+     * we would want lening treatment for floating point numbers in sql-spec tests where the comparison is being made with H2.
```

Comment:
is `lening` a correct word?

Reply:
It should be `lenient` :-). Will change.